src/clojurebridge_turtle/core.clj

```diff
@@ -43,11 +43,12 @@
   ([name color]
      (let [n (keyword name)]
        (when-not (n @turtles)
-         (swap! lines assoc n [])
-         (swap! turtles assoc n {:x 0
-                                 :y 0
-                                 :angle 90
-                                 :color color}))
+         (dosync
+          (swap! lines assoc n [])
+          (swap! turtles assoc n {:x 0
+                                  :y 0
+                                  :angle 90
+                                  :color color})))
        {n (n @turtles)})))
```

Comment:
@brandonbloom deleted `dosync` at the pull request https://github.com/ClojureBridge/welcometoclojurebridge/pull/3 . `dosync` does some trick?